scripts/ci-build.sh

```diff
@@ -19,5 +19,4 @@ set -o nounset
 set -o pipefail
 
 REPO_ROOT=$(dirname "${BASH_SOURCE}")/..
-
-go build  "${REPO_ROOT}"/...
+cd "${REPO_ROOT}" && make build
```

Comment:
I actually like the idea of using the go build command here, as it will catch build paths that aren't listed in the makefile.